### ISSUE TITLE
PR Merged into Master Branch without Reviews

### ISSUE THREAD

**Issue by lussierc** [Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment#2]
It seems that Pull Request #1 was merged into `master` without any reviews or approvals. We should revert these changes in master.

**Comment by gkapfham** [comment]
Hello @lussierc, can you comment on whether or not this issue was resolved? If it was resolved correctly then could I trouble you to please close the issue? Thanks! If there are ways in which I can help to resolve the issue, please let me or one of the technical leaders or project managers know.

**Comment by lussierc** [comment]
Hi @gkapfham, I created Pull Request #3 that would resolve this issue but it has not been merged into the `master` branch. 

With this, I was wondering what the standard is for situations like this? Since I have made a PR that fixes the Issue but it has not been merged would I still close the issue? Or, would I wait to close the issue once the PR has been merged into `master` or via an automatic closure by mentioning the issue in the PR?

**Comment by enpuyou** [comment]
@lussierc As long as you referenced the issue in the pull request, it will automatically be closed via the merge of the PR. This can probably answer part of your question. But I do see that there might be circumstances that issues remain open after PR being merged.

**Comment by lussierc** [comment]
@enpuyou I did mention the issue in the PR originally so it should be automatically closed once the PR is merged into master.

**Comment by lussierc** [comment]
As a pull request (PR #7) has now been made for the work discussed in this Issue, I am closing the Issue.

Any further discussion about this section should continue in the aforementioned PR.

**Comment by enpuyou** [comment]
Closing this issue as https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment/pull/3 has been merged
